tools/decode-bounds-reference.py

```
@@ -0,0 +1,192 @@
+        "field_notes": {
+            "construction": "input = bytes.fromhex(repeat_hex) * count + bytes.fromhex(suffix_hex)",
+            "nesting_depth": "collection headers along the deepest spine (str/bin count as 0)",
+            "declared_slots": "sum of every header's declared element/byte count (a nested header counts as one element of its parent)",
+            "reject_reasons": "which rule(s) the vector violates; a maintainer note, not a normative message",
+        },
+        "reject_vectors": reject,
+        "accept_vectors": accept,
+    }
+
+
+def check(condition: bool, name: str, detail: str) -> None:
```

Ruff (0.16.3)

[warning] Boolean-typed positional argument in function definition

(FBT001)